src/main/java/de/theit/jenkins/crowd/CrowdSecurityRealm.java

```diff
@@ -326,6 +326,10 @@ public Integer getCacheTTL() {
         return cache == null ? null : cache.getTtl();
     }
 
+    public boolean getUseTokenCache() {
+        return cache == null ? false : cache.getUseTokenCache();
+    }    
```

Comment:
for public api, please add an proper documentation
but i think it should be removed.